packages/format-library/src/link/inline.js

```diff
@@ -63,14 +63,24 @@ function InlineLinkUI( {
 		};
 	}, [] );
 
-	const linkValue = {
-		url: activeAttributes.url,
-		type: activeAttributes.type,
-		id: activeAttributes.id,
-		opensInNewTab: activeAttributes.target === '_blank',
-		nofollow: activeAttributes.rel?.includes( 'nofollow' ),
-		title: richTextText,
-	};
+	const linkValue = useMemo(
+		() => ( {
+			url: activeAttributes.url,
+			type: activeAttributes.type,
+			id: activeAttributes.id,
+			opensInNewTab: activeAttributes.target === '_blank',
+			nofollow: activeAttributes.rel?.includes( 'nofollow' ),
+			title: richTextText,
+		} ),
+		[
+			activeAttributes.id,
+			activeAttributes.rel,
+			activeAttributes.target,
+			activeAttributes.type,
+			activeAttributes.url,
+			richTextText,
+		]
+	);
```

Comment:
We should document this pattern in the `README` for the component.

Reply:
We could probably mention value memoization somewhere.

Reply:
Please see https://github.com/WordPress/gutenberg/pull/54659

Reply:
Thank you, @getdave!